code/modules/events/blob.dm

```diff
@@ -3,7 +3,7 @@
 	typepath = /datum/round_event/ghost_role/blob
 	weight = 10
 	max_occurrences = 1
-
+	earliest_start = 25 MINUTES

```

Comment:
Blobs is kinda screwed if they must always spawn this late. Rest seems fine though

Reply:
Do you have a suggestion of a specific time?

Reply:
I'll just lower it to 20mins.